client_webhooks.go

```diff
@@ -0,0 +1,23 @@
+package monta
+
+import (
+	"context"
+	"net/url"
+)
+
+// GetWebhookConfigResponse is the response output from the [Client.GetWebhookConfig] method.
+type GetWebhookConfigResponse struct {
+	// A HTTPS URL to send the webhook payload to when an event occurs.
+	WebhookURL string `json:"webhookUrl"`
+	// A cryptoghrapic secret used to sign the webhook payload.
+	WebhookSecret string `json:"webhookSecret"`
+	// A list of event type to subscribe to. Use of ["*"] means subscribe to all.
+	EventTypes []*string `json:"eventTypes"`
+}
+
+// GetWebhookConfig to get your webhook config.
+func (c *clientImpl) GetWebhookConfig(ctx context.Context) (*GetWebhookConfigResponse, error) {
+	path := "/v1/webhooks/config"
```

Comment:
Will we use this or similar paths in other contexts? If so it might make sense to extract it to make it easier to reuse it

Reply:
The path was extracted in another PR 👍